docs/ionq/jobs.md

```diff
@@ -0,0 +1,120 @@
+```
+Which results in
+```
+x=0000000000000000000000000000000000000000000000000000111111111111111111111111111111111111111111111111
+```
+Looking at these results you should notice something strange. What are the odds
+that the x measurements were all 0s followed by all 1s?  The reason for this
+sorting is that the IonQAPI only returns statistics about the results, i.e. what
+count of results were 0 and what count were 1 (or if you are measuring
+multiple qubits the counts of the different outcome bit string outcomes).  In
+order to make this compatible with Cirq's notition of `cirq.Result`, these
+are then converted into raw results with the exactly correct number of
+results (in lexical order). In other words, the measurement results are not
+in an order corresponding to the temporal order of the measurements.
+
+When calling run, one will need to include the number of `repetitions` or shots
+for the given circuit.  In addition, if there is no `default_target` set on the
+service, then a `target` needs to be specified.  Currently the supported targets
+are `qpu` and `simulator`.
+
+### Via a sampler
+
+Another method to get results from the IonQ API is to use a sampler.  A sampler
+is specifically design to be a lightweight interface for obtaining results
+in a [pandas](https://pandas.pydata.org/) dataframe and is the interface
+used by other classes in Cirq for objects that process data.  Here is a
+simple example showing how to get a sampler and use it.
+
+```python
+qubit = cirq.LineQubit(0)
+circuit = cirq.Circuit(
+    cirq.X(qubit)**0.5,            # Square root of NOT.
+    cirq.measure(qubit, key='x')   # Measurement store in key 'x'
+)
+sampler = service.sampler(target='qpu')
+result = sampler.run(program=circuit, repetitions=100)
+print(result)
+```
+
+### Via create job
+
+The above two methods, using run and the sampler, both block waiting for
+results.  This can be problematic when the queueing time for the service
+is long.  Instead it is recommended that one use the job api directly.
+In this pattern one first create the job with the quantum circuit one
+wishes to run, and the service immediately returns an object that has
+the id of the job.  This job id can be recorded and at any time in
+the future one can query for the results of this job.
+
+```python
+qubit = cirq.LineQubit(0)
+circuit = cirq.Circuit(
+    cirq.X(qubit)**0.5,            # Square root of NOT.
+    cirq.measure(qubit, key='x')   # Measurement store in key 'x'
+)
+job = service.create_job(circuit=circuit, target='qpu', repetitions=100)
+print(job)
+```
+which shows that the returned object is a `cirq.ionq.Job`:
+```
+cirq.ionq.Job(job_id=93d111c1-0898-48b8-babe-80d182f8ad66)
+```
+
+One difference between this approach and the run and sampler methods
+is that the returned job objects results are more directly related to the
+return data from the IonQ API.  They are of types `ionq.QPUResult` or
+`ionq.SimulatorResult`.  If one wishes to massage these into the
```

Comment:
`one wishes` - > `you wish`

Reply:
you all are cramping my colloquial style.  Changed.